src/schema/v2/location.ts

```diff
@@ -119,9 +123,34 @@ export const LocationType = new GraphQLObjectType<any, ResolverContext>({
   }),
 })
 
-const Location: GraphQLFieldConfig<void, ResolverContext> = {
+export const Location: GraphQLFieldConfig<void, ResolverContext> = {
   type: LocationType,
   description: "A Location",
 }
 
-export default Location
+export const LocationsEdgeInterface = new GraphQLInterfaceType({
+  name: "LocationsEdgeInterface",
+  fields: {
+    node: {
+      type: LocationType,
+    },
+    cursor: {
+      type: GraphQLString,
+    },
+  },
+})
+
+export const LocationsConnectionInterface = new GraphQLInterfaceType({
+  name: "LocationsConnectionInterface",
+  fields: {
+    pageCursors: { type: new GraphQLNonNull(PageCursorsType) },
+    pageInfo: { type: new GraphQLNonNull(PageInfoType) },
+    edges: { type: new GraphQLList(LocationsEdgeInterface) },
+  },
+})
+
+export const locationsConnection = connectionWithCursorInfo({
+  nodeType: LocationType,
+  connectionInterfaces: [LocationsConnectionInterface],
+  edgeInterfaces: [LocationsEdgeInterface],
+})
```

Comment:
Are you really in need of these interfaces in client code?

Reply:
Opting to remove these interfaces.